engines/scumm/smush/smush_player.cpp

```diff
@@ -669,7 +669,9 @@ void SmushPlayer::handleTextResource(uint32 subType, int32 subSize, Common::Seek
 }
 
 const char *SmushPlayer::getString(int id) {
-	return _strings->get(id);
+	if (_strings != nullptr) {
+		return _strings->get(id);
+	} else return nullptr;
```

Comment:
We could have a warning in here too, but not necessarily, as it could be a little bit redundant. Anyhow I would personally format the `else` like this:
```c++
} else {
    return nullptr;
}
```

Just a matter of taste though 😄

Reply:
Done and done. I added the warning here too, since in today's testing with all TRS files removed, this is the case that was triggering during the bike fight.